liblouis/lou_translateString.c

```diff
@@ -2587,7 +2587,7 @@ resolveEmphasisWords(unsigned int *buffer, const EmphasisClass class,
 
 					/* if whole word is one symbol,
 					 * turn it into a symbol */
-					if (class.symbol && word_start + 1 == i)
```

Comment:
How do you know this is always true?

Reply:
`class.symbol` is never 0. It's always a "1" shifted by a certain number.

Reply:
(it's a constant)

Reply:
OK, weird. Was it a constant before?

Reply:
Yeah, sure. I didn't change anything. Just refactoring. I don't know what's Mike intention with this was. Maybe some remnant of before when it was not constant.